src/component/button/HsiButton/HsiButton.tsx

```diff
@@ -256,7 +256,8 @@ export class HsiButton extends React.Component<HsiButtonProps, HsiButtonStatePro
       shape,
       iconName,
       tooltip,
-      tooltipPlacement
+      tooltipPlacement,
+      ...passedThroughProps
```

Comment:
This requires to filter out all custom non-button props, e.g. `drillDown`.

Reply:
Nice catch! Added them in e15d1cc